### PR TITLE
Feature/#15732 review and complete official services documentation

### PR DESCRIPTION
**Pull request by agreggio-slingr**
## Summary

Service documentation

### CODE REVIEW

**Review by jguardiolagarcia — COMMENTED** [review]
image
We are still able to see endpoints docs. Is that okey according to what you talked with seba? I can do the fix if you want

File: content/en/extensions/getting_started/overview.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Some external services will also provide a webhook URL where you can receive an asynchronous response from your service.
+These can be handled using service listeners so the app can act upon a received response.
+[You can find more information on how to use services here]({{<ref "/dev-reference/data-model-and-logic/services.md#endpoints-usage" >}}).
+
+## **Service configuration**
+
+Services can be added to an app in the app builder, in the **`Model > Services`** section. Here, you can
+set the configuration of services, which basically consists of the type of service, name, and some
+legacy-service-specific settings like API keys, users, passwords, etc.
```
Comment (pasaperez-slingr) [inline content/en/extensions/getting_started/overview.md:55]: I would not add references to legacy services, as they are likely to disappear once deprecated.

File: content/en/extensions/services/common.md [hunk 2]
```diff
@@ -0,0 +1,1039 @@
...
+
+## **Scripts**
+
+You need to configure scripts in the descriptor file:
+
+
+```
+...
+"scripts":[
+    "sample.js",
```
Comment (pasaperez-slingr) [inline content/en/extensions/services/common.md:860]: We use another path to define in services

**Review by pasaperez-slingr — APPROVED** [review]

**Comment by agreggio-slingr** [comment]
> image We are still able to see endpoints docs. Is that okey according to what you talked with seba? I can do the fix if you want

I still don't know if we need to delete the documentation for endpoints.

**Pull request merged**